src/Helper/HasherHelper.php

```diff
@@ -79,11 +85,11 @@ public function writeWithFinder(Finder $finder, FileHashStrategy $strategy = Fil
         foreach ($finder as $file) {
             switch ($strategy) {
                 case FileHashStrategy::Content:
-                    hash_update_file($this->hashContext, $file->getPathname());
+                    hash_update_file($this->hashContext, $file->getRealPath());
```

Comment:
Why did you change that?

Reply:
For coherence with `writeFile()`, `writeGlob()` with usage of `realpath()`

Relative path should not be used with usage of `$global`